app/Http/Controllers/SiteController.php

```diff
@@ -0,0 +1,284 @@
+<?php
+namespace App\Http\Controllers;
+
+use App\Http\Models\PublishedContract;
+use App\Http\Services\APIService;
+use Illuminate\Http\Request;
+use App\Http\Models\Subscriber;
+use App\Services\ConfirmationService;
+
+/**
+ * Class SiteController
+ * @property APIService api
+ * @package App\Http\Controllers
+ */
+class SiteController extends Controller
+{
+    public function __construct(APIService $api)
+    {
+        $this->api = $api;
+    }
+
+    /**
+     * Home Page
+     *
+     */
+    public function home()
+    {
+        $groups = $this->api->corporate_group();
+        asort($groups);
+
+        $countries = $this->getCountryList();
+
```

Comment:
remove extra line
and change to this
$email = $request->get('email');
$email = $request->get("source"');